src/Uno.UI/UI/Xaml/Internal/ContentRoot.cs

```diff
@@ -68,6 +69,10 @@ public ContentRoot(ContentRootType type, Color backgroundColor, UIElement? rootE
 			FocusAdapter = new FocusAdapter(this);
 			FocusManager.SetFocusObserver(new FocusObserver(this));
 
+			CompositionTarget = new CompositionTarget();
+			CompositionTarget.Root = VisualTree.RootElement.Visual;
```

Comment:
To make this more concise, I think the `Root` setter could automatically set the `CompositionTarget` as well, thus the following line's logic would happen in the setter